### ISSUE TITLE
app file: preview not working sometime when you navigate in the preview

### ISSUE THREAD

**Issue by PhilippeAccorsi** [tracim/tracim#1118]
At present

Sometime when you navigate in page of previw you can see this error. If you refresh the page the preview work correctly

~~~
14:55:35 INFO  [root:30][waitress] New Preview builder of class<class 'preview_generator.preview.builder.plain_text.PlainTextPreviewBuilder'>
14:55:42 INFO  [root:105][waitress] Removing temporary copy file /tmp/tracim/preview/d7c2740c3a15929efdc35ded00d80a48-500x500-page8.bat
14:55:42 INFO  [hapic:244][waitress] Exception FileNotFoundError occured, return 500 http_code : [Errno 2] No such file or directory: '/tmp/tracim/preview/d7c2740c3a15929efdc35ded00d80a48-500x500-page8'
14:55:42 DEBUG [hapic:246][waitress] Traceback (most recent call last):
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/tweens.py", line 41, in excview_tween
    response = handler(request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/router.py", line 148, in handle_request
    registry, request, context, context_iface, view_name
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/view.py", line 657, in _call_view
    response = view_callable(context, request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/viewderivers.py", line 325, in secured_view
    return view(context, request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/viewderivers.py", line 401, in viewresult_to_response
    result = view(context, request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/pyramid/viewderivers.py", line 95, in mapped_view
    return _mapped_view(context, request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/hapic.py", line 128, in wrapper
    return func(*args, **kwargs)
  File "/home/algoo/repo/tracim_v2/backend/tracim_backend/lib/utils/authorization.py", line 142, in wrapper
    return func(self, context, request)
  File "/home/algoo/repo/tracim_v2/backend/tracim_backend/lib/utils/authorization.py", line 184, in wrapper
    return func(self, context, request)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 615, in _execute_wrapped_function
    func_kwargs,
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 615, in _execute_wrapped_function
    func_kwargs,
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 615, in _execute_wrapped_function
    func_kwargs,
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 89, in wrapper
    response = self._execute_wrapped_function(func, args, kwargs)
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/hapic/decorator.py", line 100, in _execute_wrapped_function
    return func(*func_args, **func_kwargs)
  File "/home/algoo/repo/tracim_v2/backend/tracim_backend/views/contents_api/file_controller.py", line 514, in sized_preview_jpg
    width=hapic_data.path.width,
  File "/home/algoo/repo/tracim_v2/backend/tracim_backend/lib/core/content.py", line 1046, in get_jpg_preview_path
    file_ext=file_extension,
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/preview_generator/manager.py", line 186, in get_jpeg_preview
    mimetype=mimetype
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/preview_generator/preview/builder/document_generic.py", line 84, in build_jpeg_preview
    _file, file_extension, cache_path, output_path, mimetype
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/preview_generator/preview/builder/office__libreoffice.py", line 50, in _convert_to_pdf
    cache_path, output_filepath, mimetype
  File "/home/algoo/repo/tracim_v2/backend/env/lib/python3.6/site-packages/preview_generator/preview/builder/office__libreoffice.py", line 108, in convert_office_document_to_pdf
    with open(output_filepath, 'rb') as pdf_handle:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tracim/preview/d7c2740c3a15929efdc35ded00d80a48-500x500-page8'

14:55:42 INFO  [root:105][waitress] Removing temporary copy file /tmp/tracim/preview/d7c2740c3a15929efdc35ded00d80a48-1920x1080-page9.bat
~~~

Expected

- [ ] preview work all the time without error

**Comment by inkhey** [comment]
should be fixed with preview_generator 0.9(https://github.com/algoo/preview-generator/pull/56) and tracim requirement update in https://github.com/tracim/tracim_v2/pull/125

**Comment by PhilippeAccorsi** [comment]
Verified.
